dotCMS/src/main/webapp/html/css/dijit-dotcms/dotcms.css

```diff
@@ -5043,13 +5043,15 @@
   margin-top: 0.5rem;
 }
 .dijitComboBoxMenuPopup .dijitMenuItem {
+  padding: 0 0.75rem;
+  color: #14151a;
```

Comment:
Is there a variable for this color?

Reply:
This file is the build, it does not have variables, it just generates the css using the homologate .scss file